src/zope/i18n/testing.py

```diff
@@ -36,8 +39,22 @@ def setUp(test=None):
     zope.component.provideAdapter(BrowserLanguages)
 
 
-class PlacelessSetup(object):
+class PlacelessSetup(CleanUp):
```

Comment:
It looks like this change caused a regression: see https://github.com/zopefoundation/zope.i18n/issues/30.

I don't entirely understand why.